Examples/Swift/AnnotationViewMultipleExample.h

```diff
@@ -0,0 +1,13 @@
+//
+//  AnnotationViewMultipleExample.h
+//  Examples
+//
+//  Created by Nadia Barbosa on 12/13/16.
+//  Copyright © 2016 Mapbox. All rights reserved.
+//
+
```

Comment:
I’m guessing it’s not necessary — that file already exists [in the Objective-C folder.](https://github.com/mapbox/ios-sdk-examples/blob/b4e5a29afed1e691615913aa6d4d4e247e1e4b3f/Examples/ObjectiveC/AnnotationViewMultipleExample.h)